addons/survey/models/survey_survey.py

```diff
@@ -506,16 +509,16 @@ def action_survey_user_input_invite(self):
         return action
 
     @api.multi
-    def has_attempts_left(self, partner, email):
+    def has_attempts_left(self, partner, email, invite_token):
```

Comment:
Maybe, we can remove `partner` and `email`, and only use `invite_token`(required param), as we now want to count attempt by pool (identified by invite_token).

Reply:
Not necessary anymore (as discussed).